web/src/main/kotlin/no/nav/su/se/bakover/web/Application.kt

```diff
@@ -204,6 +208,7 @@ internal fun Application.susebakover(
         level = Level.INFO
         filter { call ->
             if (call.request.httpMethod.value == "OPTIONS") return@filter false
+            if (call.pathShouldBeExcluded(naisPaths + personPath)) return@filter false
```

Comment:
Nå som vi bare har to, kanskje vi kunne knekt de ut over to linjer (den + syntaksen stusset jeg ganske mye på selv).

```suggestion
            if (call.pathShouldBeExcluded(naisPaths)) return@filter false
            if (call.pathShouldBeExcluded(personPath)) return@filter false
```

Reply:
Enig.